site/content/en/docs/Guides/feature-stages.md

```diff
@@ -31,6 +31,7 @@ The current set of `alpha` and `beta` feature gates are:
 | [Player Tracking]({{< ref "/docs/Guides/player-tracking.md" >}}) | `PlayerTracking` | Disabled | `Alpha` | 1.6.0 |
 | [SDK Send GameServer on Watch execution]({{< ref "/docs/Guides/Client SDKs/_index.md#watchgameserverfunctiongameserver" >}}) | `SDKWatchSendOnExecute` | Disabled | `Alpha` | 1.7.0 |
 | Fix for RollingUpdate [Scale down](https://github.com/googleforgames/agones/issues/1625) and additional [details]({{< ref "/docs/Guides/fleet-updates.md#alpha-feature-rollingupdateonready" >}}) | `RollingUpdateOnReady` | Disabled | `Alpha` | 1.9.0 |
+| [Utilize Node ExternalDNS](https://github.com/googleforgames/agones/issues/1921) and additional [details]({{< ref "/docs/Guides/utilize-node-externaldns.md" >}}) | `NodeExternalDNS` | Disabled | `Alpha` | 1.11.0 |
```

Comment:
We'll need to wrap this new entry in a `feature` shortcode.

You an see an example here: https://agones.dev/site/docs/contribute/#within-a-page

Since 1.110.0 RC is tomorrow (I expect this won't land in this release, just because timing), this will probably need to target 1.12.0 - but we can adjust later if needed.